awsproviderlint/passes/AWSAT003/testdata/src/a/main.go

```diff
@@ -0,0 +1,52 @@
+package a
+
+import (
+	"fmt"
+)
+
+func f() {
+	/* Passing cases */
```

Comment:
It might be nice to show a passing example using one of the [`endpoints` package regional constants](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/#pkg-constants), which could be an easy way to bypass the linter failure in normal code. e.g.

```
endpoints.UsEast1RegionID
```